requirements/deps/constraints.txt

```diff
@@ -1,7 +1,8 @@
 ####################################################################################################
 # This file can house global constraints that aren't *direct* requirements of the package or any
 # extras. Putting a dependency here will only affect dependency sets that contain them -- in other
-# words, if something does not require a constraint, it will not be installed.
+# words, if something does not require a constraint, it will not be installed. Refer to the "Adding
+# a new Dependency Group" notes in Notion for more info about managing requirements.
```

Comment:
Open-source users aren't going to have access to the internal Notion. Not sure exactly what to do about that but something to think about.

Reply:
That did cross my mind and I was wondering if we should just add a README or something with the contents of that Notion page. It's just general info that can be a good reference, even for potential contributors. Lmk if you have any more thoughts or if we should bring someone else to weigh in

Reply:
I'd ask Matt and let him make the call.

Reply:
@MthwRobinson ?

Reply:
I'd leave references to our internal Notion out of the open source repo.